templates/etc/quagga/Quagga.conf.j2

```diff
@@ -4,6 +4,7 @@ hostname {{ quagga_hostname }}
 password {{ quagga_password }}
 enable password {{ quagga_enable_password }}
 service password-encryption
+service integrated-vtysh-config
```

Comment:
Is this the Cumulus default?  Quagga docs [recommend against using this](http://www.nongnu.org/quagga/docs/docs-multi/VTY-shell-integrated-configuration.html), although I can see arguments for either way.